main.go

```diff
@@ -37,17 +47,30 @@ func main() {
 		Name("Sandy").
 		Build()
 
+	// Set featureFlagKey to the feature flag key you want to evaluate.
+	var featureFlagKey = "my-boolean-flag"
+
+	if os.Getenv("LAUNCHDARKLY_FLAG_KEY") != "" {
+		featureFlagKey = os.Getenv("LAUNCHDARKLY_FLAG_KEY")
+	}
+
 	flagValue, err := ldClient.BoolVariation(featureFlagKey, context, false)
 	if err != nil {
 		showMessage("error: " + err.Error())
 	}
 
-	showMessage(fmt.Sprintf("Feature flag '%s' is %t for this context", featureFlagKey, flagValue))
+	showMessage(fmt.Sprintf("The '%s' feature flag evaluates to %t.", featureFlagKey, flagValue))
+
+	if flagValue {
+		showBanner()
+	}
 
-	// Here we ensure that the SDK shuts down cleanly and has a chance to deliver analytics
-	// events to LaunchDarkly before the program exits. If analytics events are not delivered,
-	// the context attributes and flag usage statistics will not appear on your dashboard. In
-	// a normal long-running application, the SDK would continue running and events would be
-	// delivered automatically in the background.
-	ldClient.Close()
+	updateCh := ldClient.GetFlagTracker().AddFlagValueChangeListener(featureFlagKey, context, ldvalue.Null())
+
+	for event := range updateCh {
```

Comment:
We should have a bool that holds the last value, and only output the message/banner if it changed from the last value.

Reply:
I said this because it's what Matthew did on the Rust update, but now I'm not so sure. It'd only print out a redundant msg if the targeting rules changed, I think, which would be unlikely in this scenario?

Might be missing something.. I thought there was another reason.